### PR TITLE
[Gear] Neltharion's Call to Dominance

### PR DESCRIPTION
**Pull request by Azevara**

### CODE REVIEW

**Comment by Azevara** [comment]
Not merging this quite yet even if checks pass - callbacks are not being triggered for some Summon spells and investigation of that must be completed in case further changes are needed.

File: engine/player/unique_gear_dragonflight.cpp [hunk 1]
```diff
@@ -3886,6 +3886,111 @@ void neltharions_call_to_chaos( special_effect_t& effect )
...
+      driver_id = 408262;
+      proc_spell_id = { { 201430, 360952 } }; // Stampede or Coordinated Assault
+      break;
+    case HUNTER_MARKSMANSHIP:
+      driver_id = 408262;
+      proc_spell_id = { { 201430 } }; // Stampede
+      break;
+    case HUNTER_BEAST_MASTERY:
+      driver_id = 408262;
+      proc_spell_id = { { 201430, 120679 } }; // Stampede or Dire Beast. TOCHECK: Dire Command?
```
Comment (Pewtro) [inline engine/player/unique_gear_dragonflight.cpp]: Dire Command does indeed proc it, and proc will override any ongoing one even if the ongoing one is a higher stack value.

File: engine/player/unique_gear_dragonflight.cpp [hunk 2]
```diff
@@ -3886,6 +3886,111 @@ void neltharions_call_to_chaos( special_effect_t& effect )
...
+      driver_id = 408259;
+      proc_spell_id = { { 51533 } }; // Feral Spirit
+      break;
+    case SHAMAN_RESTORATION:
+      driver_id = 408259;
+      proc_spell_id = { { 98008, 108280, 16191 } }; // Spirit Link, Healing Tide, Mana Tide totems
+      break;
+    case MONK_BREWMASTER:
+      driver_id = 408260;
+      proc_spell_id = { { 132578 } }; // Invoke Niuzao
```
Comment (Hinalover) [inline engine/player/unique_gear_dragonflight.cpp]: Call to Arm's Invoke Niuzao is proccing the buff. 
Trigger spell: Weapons of Order: 387184 
Call to Arms talent: 397251, 
Buff spell that summons Niuzao: 395267

**Comment by Azevara** [comment]
Items not resolved by this PR:
Hunter:
Dire Beast is currently flagged with harmful=false
Dire Command doesn't have a way to trigger callbacks atm
Stampede works

Shaman:
Fire and Storm Elementals have harmful=false
Feral Spirit has harmful=false
Resto was not tested

Monk:
Invoke Niuzao and Xuen has harmful=false
Fury of Xuen doesn't have a way to trigger callbacks atm
MW was not tested

**Pull request merged**